pkg/operator/workloads/workflow.go

```diff
@@ -113,7 +113,7 @@ func Create(ctx *context.Context) (*awfv1.Workflow, error) {
 
 		manifest, err := json.Marshal(spec.Spec)
 		if err != nil {
-			return nil, errors.Wrap(err, ctx.App.Name, "workloads", spec.WorkloadID, s.ErrMarshalJSON)
+			return nil, errors.Wrap(err, ctx.App.Name, "workloads", spec.WorkloadID, configreader.ErrorMarshalJSON().Error())
```

Comment:
What do you think about moving `ErrorMarshalJSON()` to our `json` package? What would we call our `json` package when we import it?